contracts/truefi2/CrvBaseRateOracle.sol

```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.6.10;
+
+import {SafeMath} from "@openzeppelin/contracts/math/SafeMath.sol";
+import "../truefi/interface/ICurve.sol";
+
+// prettier-ignore
+contract CrvBaseRateOracle {
+    using SafeMath for uint256;
+    uint8 public constant BUFFER_SIZE = 7;
+
+    ICurve public curve;
```

Comment:
Don't mix storage variables with compile-time structs, move this below HistoricalRatesBuffer